packages/client/utils/retroGroup/updateClonePosition.ts

```diff
@@ -24,18 +51,61 @@ const getTransition = (isClipped: boolean, timeRemaining: number) => {
     : transition
 }
 
+export const getSpotlightAnimation = (
+  element: HTMLDivElement,
+  targetId: string | null | undefined,
+  groupIdsInSpotlight: string[],
+  isClose: boolean,
+  lastZIndex: number | undefined
+) => {
+  const spotlightEl = document.getElementById('spotlight')
+  const isTargetInSpotlight = targetId && groupIdsInSpotlight.includes(targetId)
+  const isInSpotlight = spotlightEl && spotlightEl.contains(element)
```

Comment:
`contains` is a DOM method so it's a little expensive (at least on mobile devices).

can we write something like
```ts
const isInSpotlight = spotlightEl?.contains(element) ?? false
const notInSpotlight = spotlightEl ? !isInSpotlight : false
```